packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -324,9 +324,29 @@ When('I mock {string} event', (eventName: string) => {
     throw new Error('Hub is not available on the window.');
   }
 
-  Hub.dispatch('auth', { event: eventName });
+  Hub.dispatch('auth', { event: eventName, data: {} });
```

Comment:
This dispatch was failing since it wasn't sending data, I updated it.

Reply:
Good catch!

Reply:
Is there a way to use type to catch it?